omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -202,14 +253,72 @@ code from the project's GitHub account to build locally::
     :doc:`/developers/build-system`
         Developer documentation page on how to build the OMERO.server
 
+3. Once the build completes, the OMERO server build output will be located in :file:`~/Projects/Omero/code/projects/openmicroscopy/dist`.
+   Prepend the :file:`bin` directory to your :envvar:`PATH`::
+
+    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
 
+   and follow the steps for setting up the database and OMERO data directory as mentioned in the previous section.
 
-Then prepend the development :file:`bin` directory to your :envvar:`PATH` to
-pick the right executbale::
+4. Activate the Virtualenv enviornment that we created earlier in the "Requirements"
+   section::
 
-    $ export PATH=~/code/projects/openmicroscopy/dist/bin:$PATH
+    $ source ~/Virtual/Omero/bin/activate
+
+5. Install Python dependencies using pip::
+
+    $ pip install -r ~Omero/server/share/web/requirements-py27-all.txt
```

Comment:
This line is not working for me.
```
omero) snapper:openmicroscopy hudson$ pip install -r ~Omero/server/share/web/requirements-py27-all.txt
Could not open requirements file: [Errno 2] No such file or directory: '~Omero/server/share/web/requirements-py27-all.txt
```

Reply:
this path to the file needs to be corrected

Reply:
In the end, the command
``/Projects/Omero/code/openmicroscopy/dist/share/web/requirements-py27-all.txt``
worked for me, but note that we did some moving of directories. (see also https://github.com/openmicroscopy/ome-documentation/pull/1766/files#r140265914)